JDI.Light/JDI.Light/Factories/UIElementFactory.cs

```diff
@@ -40,6 +40,7 @@ public static IBaseUIElement CreateInstance(Type t, By locator, IBaseElement par
                         instance.Parent = parent;
                         return instance;
                     }
+                    default: break;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty 'default' clause.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5599701)